cmd/ping/main.go

```diff
@@ -21,19 +21,28 @@ import (
 	"net/http"
 	"os"
 	"sync"
+	"context"
 
 	"github.com/GoogleCloudPlatform/gcping/internal/config"
 )
 
 var once sync.Once
 
 func main() {
+	ctx := context.Background()
 	port := os.Getenv("PORT")
 	if port == "" {
 		port = "8080"
 	}
 	log.Printf("Serving on :%s", port)
 
+	var AllEndpoints map[string]config.Endpoint 
```

Comment:
This line can be removed with the `:=` below.

Also, use `allEndpoints` or something shorter.